src/pages/Account/Account.types.ts

```diff
@@ -0,0 +1,40 @@
+import { type ChainId } from '@swapr/sdk'
+
+import { type BridgeList } from '../../services/EcoBridge/EcoBridge.types'
+import { type BridgeTransactionLog } from '../../state/bridgeTransactions/types'
+
+export enum TransactionSwapTypes {
+  Swap = 'Swap',
+}
+
+export enum TransactionBridgeTypes {
+  Bridge = 'Bridge',
+}
```

Comment:
you can create one enum for this and then sue it inside `SwapTransaction` and `BridgeTransaction`
```ts
export interface SwapTransaction {
  type: TransactionType.Swap
  from: Token
  to: Token
  summary: string
  addedTime: number
  confirmedTime?: number
  hash: string
  status: string
  network?: ChainId
}

export interface BridgeTransaction extends Omit<SwapTransaction, 'summary' | 'type' | 'addedTime'> {
  type: TransactionType.Bridge
  pendingReason?: string
  bridgeId: BridgeList
  logs: BridgeTransactionLog[]
}
```

Reply:
There are many swap types like below. So thought of using keyof typeof enum in future. :) But for now we can do this way.

`Swap, Add, Wrap, Unwrap, Remove, Approve, Claim , Withdraw, Stake, Convert, Create`